apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -58,47 +56,29 @@ export function updateDefaultList(listData) {
 }
 
 // Returns the metadata of the list of default sprites in SpriteLab in English
-export function getDefaultListMetadata() {
-  return fetch('/api/v1/animation-library/default-spritelab-metadata')
+export function getDefaultListMetadata(environment = 'production') {
+  return fetch(
+    `/api/v1/animation-library/default-spritelab-metadata/${environment}`
+  )
     .then(response => response.json())
     .catch(err => {
       return Promise.reject(err);
     });
 }
 
-export function createDefaultSpriteMetadata(listData) {
```

Comment:
This was part of the old API that we moved away from so we consolidated this work into 'regenerateDefaultSpriteMetadata'.